test/check-application

```diff
@@ -994,13 +994,26 @@ class TestFiles(testlib.MachineCase):
         b.click("button.pf-m-primary")
         b.wait_in_text("[data-item='newdir1'] .item-owner", "testuser")
 
+        # Open folder in terminal from context menu
+        b.mouse("[data-item='newdir1']", "contextmenu")
+        b.click(".contextMenu button:contains('Open in terminal')")
```

Comment:
This needs to become conditional on TEST_OS, i.e. `m.image` here. Some distros like ubuntu-2204 or RHEL 8 will *never* get this new version, and for others it will still take a fair while. A static list will be too inconvenient in the next weeks, so let's do this dynamically. We have a helper for that, and we know that Cockpit 329 (to be released today) will support it:

```py
if self.system_before(329):
    # check that the menu entry is *not* present, similar to the directory case below
else:
    # this existing check
```